frida_tools/tracer.py

```diff
@@ -345,15 +345,39 @@ def _create_resolver_script(self):
     return workingSet;
 }
 
+function parseExportsFunctionPattern (pattern) {
+    var m;
+    var f;
+    var res = pattern.split ('!');
+    
+    if (res.length == 1) {
```

Comment:
```suggestion
    if (res.length === 1) {
```

Always use strict comparisons.